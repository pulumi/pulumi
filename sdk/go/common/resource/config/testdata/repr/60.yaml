my:name:
- secure: securevalue
- b
- c
