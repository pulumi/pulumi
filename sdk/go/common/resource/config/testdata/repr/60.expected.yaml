decrypt:
  my:name: '[{"secure":"securevalue"},"b","c"]'
paths:
  my:name:
    value:
    - secure: securevalue
    - b
    - c
    string: '[{"secure":"securevalue"},"b","c"]'
    redacted: '["[secret]","b","c"]'
    object: HRAADltdaW50ZXJmYWNlIHt9/4MCAQL/hAABEAAAZv+EKAADF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAAA6/4IfAAEGc2VjdXJlBnN0cmluZwwNAAtzZWN1cmV2YWx1ZQZzdHJpbmcMAwABYgZzdHJpbmcMAwABYw==
    secure: true
    isObject: true
    secureValues:
    - securevalue
  my:name[0]:
    value:
      secure: securevalue
    string: securevalue
    redacted: '[secret]'
    object: GBAABnN0cmluZwwNAAtzZWN1cmV2YWx1ZQ==
    secure: true
    isObject: false
    secureValues:
    - securevalue
  my:name[1]:
    value: b
    string: b
    redacted: b
    object: DhAABnN0cmluZwwDAAFi
    secure: false
    isObject: false
  my:name[2]:
    value: c
    string: c
    redacted: c
    object: DhAABnN0cmluZwwDAAFj
    secure: false
    isObject: false
